internal/resources/client.go

```diff
@@ -24,9 +34,21 @@ func NewClient(cliVersion string) ResourcesClient {
 	return ResourcesClient{cliVersion: cliVersion}
 }
 
-func (c ResourcesClient) MakeRequest(accessToken, method, path, contentType string, query url.Values, data []byte, isBeta bool) ([]byte, error) {
-	client := http.Client{}
+func (c ResourcesClient) MakeUnauthenticatedRequest(
```

Comment:
Now both clients share the same code, so we don't need to worry about drifting implementations.